simtools/simtel/simtel_config_reader.py

```diff
@@ -159,27 +168,57 @@ def export_parameter_dict_to_json(self, file_name, dict_to_write):
     def compare_simtel_config_with_schema(self):
         """
         Compare limits and defaults reported by simtel_array with schema
-        (for debugging purposes; simple printing).
+        (for debugging purposes; simple printing). Check for differences
+        in 'default' and 'limits' entries.
 
         """
 
-        self._logger.info(
-            f"Comparing simtel_array configuration with schema for {self.parameter_name} "
-            f"(sim_telarray: {self.simtel_parameter_name})"
-        )
+        for data_type in ["default", "limits"]:
+            _from_simtel = self.parameter_dict.get(data_type)
+            # ignore limits checks for boolean
+            if data_type == "limits" and self.parameter_dict.get("type") == "bool":
+                continue
+            try:
+                if data_type == "limits":
+                    _from_schema = [
+                        self.schema_dict["data"][0]["allowed_range"].get("min"),
+                        self.schema_dict["data"][0]["allowed_range"].get("max"),
+                    ]
+                    _from_schema = _from_schema[0] if _from_schema[1] is None else _from_schema
```

Comment:
This means for parameters where we only set lower limits. We don't have/expect to have upper limits?

Reply:
Some parameters have only lower limits (e.g., QE_VARIATION needs to be >=0., but could take any value; but obviously not any large value is useful)